plugin-flex-ts-template-v2/src/feature-library/park-interaction/actions/ParkInteraction.ts

```diff
@@ -71,3 +72,17 @@ export const parkInteraction = async (payload: ParkInteractionPayload) => {
     });
   }
 };
+
+export const unparkInteraction = async (payload: UnparkInteractionPayload) => {
+  try {
+    await UnparkInteractionService.unparkInteraction(payload.ConversationSid, payload.WebhookSid);
+    return Notifications.showNotification(UnparkInteractionNotification.UnparkSuccess);
+  } catch (error) {
+    const message = (error as any)?.message;
+    console.log(message);
+    if (message.status === 400) return Notifications.showNotification(UnparkInteractionNotification.UnparkError);
```

Comment:
When unparking fails due to a closed conversation, can we remove it from the Sync map?

Reply:
Done